flang/test/Lower/CUDA/cuda-kernel-loop-directive.cuf

```diff
@@ -1,4 +1,5 @@
 ! RUN: bbc -emit-hlfir -fcuda %s -o - | FileCheck %s
+! RUN: bbc -emit-hlfir -fcuda %s -o - | fir-opt | FileCheck %s
```

Comment:
Thanks for adding this here. It'll make the test more robust.

Reply:
learned form your previous PR.